convey/stack_trace_test.go

```diff
@@ -2,12 +2,23 @@ package convey
 
 import (
 	"fmt"
+	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/smartystreets/goconvey/convey/reporting"
 )
 
+var goroutineCounter = regexp.MustCompile(`goroutine \d+ \[`)
```

Comment:
`goroutineStatePattern` or `...RE` or similar? Counter implies that it's counting something to me :)

Reply:
ya, good point. It's really the function below that does the counting anyway. Switched to goroutineRE, if that's alright